gopuff/order-status
# [Feature] Add status progress bar

#### What does this PR do?
Adds the status progress bar to the order status page
 #### How should this be manually tested?
First, pull then checkout to this branch

`git pull`

`git checkout ft-OS-6-progress-bar`

Then

`yarn serve`

To see the progress changes, go to **src/services/API/fixtures/order-status.json** and 
change the **status** key value to either pack,drive or ding-dong to see the corresponding changes.
 #### Any background context you want to provide?
This PR also contains new additions like the config and services folders that basically setup constants for the project and also create a pipeline for interaction with our API(once it's ready). For now I used a fixture.
 #### What are the relevant Jira story IDs?
[OS-6](https://gopuff.atlassian.net/jira/software/projects/OS/boards/33/backlog?selectedIssue=OS-6)
 #### Screenshots (if appropriate)

`None`

 #### Questions:
`None`
